src/client/app/App.tsx

```
@@ -670,97 +812,150 @@ export const App = () => {
+                            className="text-2xl font-black italic tracking-[0.12em] text-transparent bg-clip-text uppercase"
+                            style={{ backgroundImage: 'linear-gradient(90deg, #00E5FF 0%, #4DE2FF 100%)' }}
+                        >
+                            // PILOT CONFIG
+                        </h2>
+                    </div>
+
+                    {/* Name input */}
+                    <div className="space-y-1.5">
+                        <label className="font-mono text-[9px] tracking-[0.2em] text-[#00E5FF]/40 uppercase block">
+                            CALLSIGN
+                        </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)